src/jit.rs

```diff
@@ -114,6 +114,10 @@ impl JitProgramSections {
         }
         Ok(())
     }
+
+    pub fn mem_size(&self) -> usize {
+            self.total_allocation_size
```

Comment:
In https://github.com/solana-labs/rbpf/pull/255 I removed `total_allocation_size` as that was too confusing. So now it has to be:
```Rust
let pc_loc_table_size = round_to_page_size(self.pc_section.len() * 8, self.page_size);
let code_size = round_to_page_size(self.text_section.len(), self.page_size);
pc_loc_table_size + code_size
```